Templates/BlobTrigger-Simple-ImageResize-CSharp/function.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Well, the problem is that the nuget one is not at all automated--users have to go into Kudu and manually touch project.json. Since image resizing seems like such a basic scenario, I would like customers to be able to do something easily.